alpha/scripts/utils/addPartnerToVoicebase.php

```diff
@@ -0,0 +1,111 @@
+<?php
```

Comment:
We add permissions using ini files, admin-console and alpha/scripts/utils/permissions/enablePermissionForPartner.php.
Please consider to remove this script.